content/en/docs/setup/install/helm/index.md

```diff
@@ -44,13 +44,6 @@ follow the instructions below.
     $ kubectl create namespace istio-system
     {{< /text >}}
 
-1. Install the Istio base chart which contains cluster-wide resources used by
-   the Istio control plane:
-
-    {{< text bash >}}
-    $ helm install --namespace istio-system istio-base manifests/charts/base
-    {{< /text >}}
-
     {{< warning >}}
```

Comment:
This warning isn't really part of the first step. Probably better to unindent and move to the top (to line 40).

Reply:
Done.